content/Solutions/Optimizations/Networking.md

```diff
@@ -0,0 +1,339 @@
+Implement Quality of Service policies to prioritize storage traffic over other non-essential traffic. This ensures that data transfers to and from the NAS receive the necessary priority and resources.
+
+As your network grows, isolating the NAS onto its own subnet provides a scalable solution.
+It allows you to maintain control and security even as you add more devices and services to the network.
+
+See [Managing Interfaces]({{< relref "managinginterfaces.md" >}}) and [Setting Up Static IPs]({{< relref "settingupstaticips.md" >}}) for more information.
+
+## iSCSI
+
+iSCSI shares require specific networking considerations.
+iSCSI should be its own dedicated VLAN network to isolate it from other network traffic.
+This enhances security, reduces the risk of interference, and provides easier Quality of Service (QoS) management.
+
+Prioritize iSCSI traffic over other types of traffic on the network.
+This ensures that storage-related activities receive the necessary network resources for optimal performance.
+
+Ensure consistent network configurations across all components involved in the iSCSI setup.
+Consistency helps in maintaining predictability and stability in the network environment.
+
+See [Block Shares (iSCSI)]({{< relref "scale/scaletutorials/shares/iscsi/_index.md" >}}) for more information.
+
+#### MPIO for iSCSI
+
+Use MPIO (Multipath I/O) to combine interfaces when using iSCSI.
+Do not use LAGG link aggregation.
+
+MPIO is designed to improve the reliability and performance of data transfers by using multiple physical paths between the iSCSI initiator (client) and the iSCSI target (storage).
+MPIO can distribute I/O traffic across multiple paths, balancing the load and improving overall performance.
+If one path fails, MPIO can automatically switch to an alternate path, ensuring continuous access to the storage.
+
+Both the iSCSI initiator and target must support MPIO for it to be effective.
+MPIO requires configuration on both the initiator and target sides. Each path needs to be properly identified and managed.
+
+To configure multipath for iSCSI, go to the **iSCSI** screen and configure additional Portal IP addresses.
+
+#### iSCSI and VMWare
+
+Refer to the [VMWare best practices guide](https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.storage.doc/GUID-BC516B24-7EAB-4ADA-819A-10DC496DEE9B.html) for information on configuring multiple network adapters when using VMWare with your TrueNAS system.
+
+#### Jumbo Frames
+
+Enable jumbo frames on an iSCSI share to enhance storage performance through the transmission of larger data packets, reducing overhead and improving overall efficiency.
+
+To enable jumbo frames, go to the **Network** screen and edit the parent interface(s).
+Set the **MTU** (Maximum Transmission Unit) to 9000.
+Click **SAVE**.
+
+Reboot the system and restart the iSCSI share to inherit.
+
+#### ALUA
```

Comment:
Maybe add a bit to contact support if they have questions on this so as to help validate the network design to ensure best practices or something.